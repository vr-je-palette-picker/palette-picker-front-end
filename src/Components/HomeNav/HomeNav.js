import React, { Component } from 'react';
import { Link } from 'react-router-dom';
import { IoMdRefresh } from 'react-icons/io';
import { GiSave } from 'react-icons/gi';
import { FaRegFolder } from 'react-icons/fa';
import './HomeNav.scss';

export class HomeNav extends Component {
  render() {
    const { randomColors, showModal} = this.props;
    return (
      <div className='HomeNav'>
        <h1 className='HomeNav__h1--title'>Palette Picker</h1>
        <div className='HomeNav__div--icon-container'>
          <IoMdRefresh className='HomeNav__icon' onClick={randomColors} />
            <GiSave onClick={showModal} className='HomeNav__icon' />
          <Link to='/projects'>
            <FaRegFolder className='HomeNav__icon' />
          </Link>
        </div>
      </div>
    );
  }
}
export default HomeNav;
