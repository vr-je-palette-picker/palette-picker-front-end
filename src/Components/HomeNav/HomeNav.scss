.HomeNav {
  display: flex;
  height: 15vh;
  justify-content: space-between;
  width: 100vw;
}
.HomeNav__h1--title {
  font-size: 60px;
  margin: 15px 10px 0px 10px;
  padding-left: 15px;
}
.HomeNav__div--icon-container {
  display: flex;
  justify-content: space-between;
  margin: 30px 30px 10px 10px;
  width: 20%;
}
.HomeNav__icon {
  height: 60px;
  width: 60px;
}
