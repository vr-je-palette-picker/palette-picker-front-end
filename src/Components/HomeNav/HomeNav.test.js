import React from 'react';
import { shallow } from 'enzyme';
import HomeNav from './HomeNav';

describe('HomeNav', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<HomeNav />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})