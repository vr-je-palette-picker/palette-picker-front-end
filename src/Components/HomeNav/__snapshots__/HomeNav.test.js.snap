// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeNav should match the snapshot 1`] = `
<div
  className="HomeNav"
>
  <h1
    className="HomeNav__h1--title"
  >
    Palette Picker
  </h1>
  <div
    className="HomeNav__div--icon-container"
  >
    <IoMdRefresh
      className="HomeNav__icon"
    />
    <GiSave
      className="HomeNav__icon"
    />
    <Link
      className="link"
      to="/projects"
    >
      <FaRegFolder
        className="HomeNav__icon"
      />
    </Link>
  </div>
</div>
`;
